client/app/scripts/utils/node-shape-utils.js

```diff
@@ -30,3 +30,7 @@ export const hexagonShapeProps = { d: curvedUnitPolygonPath(6) };
 export const heptagonShapeProps = { d: curvedUnitPolygonPath(7) };
 export const octagonShapeProps = { d: curvedUnitPolygonPath(8) };
 export const cloudShapeProps = { d: UNIT_CLOUD_PATH };
+export const cylinderShapeProps = { d: curvedUnitPolygonPath(9) };
```

Comment:
This will not create cylindrical shape, it will make another shape with **nine** sides.